bedrock/externalpages/templates/externalpages/analytics/index.html

```diff
@@ -0,0 +1,15 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "firefox/base/base-protocol.html" %}
+
+{% block content %}
+<main class="mzp-l-content">
+  <button class="mzp-c-button mzp-t-product" data-link-type="download-test" data-download-os="Desktop" data-download-location="test">download firefox</button>
```

Comment:
Suggestion (non-blocking): might it be worth adding a heading and description to this template just to explain what the page is used for? (e.g. "this page is used only for automated analytics testing" or similar?